lib/theme_check/checks/parser_blocking_javascript.rb

```diff
@@ -5,7 +5,11 @@ class ParserBlockingJavaScript < LiquidCheck
     severity :error
     category :liquid
 
-    PARSER_BLOCKING_SCRIPT_TAG = /<script(?:(?!defer|async|type=["']module['"]).)*?>/im
+    # How to read the following regex:
```

Comment:
You can use the `x` option to enable extended regexp syntax in Ruby (ignores whitespaces, so can split on multiple lines & indent), eg.:

```ruby
REGEX = %r{
  <style
  |
  display:\s*(block|inline|flex)
  |
  color:\s*\#\h+
  |
  [\w\-]+:\s*\d+(px|em)
}x
```

Then put the comments inline

Reply:
So much learning :) TY